db/migrate/20230922041845_create_join_table_hashtags_tweets.rb

```diff
@@ -0,0 +1,7 @@
+class CreateJoinTableHashtagsTweets < ActiveRecord::Migration[7.0]
+  def change
+    create_join_table :tweets, :hashtags do |t|
+      
+    end
+  end
+end
```

Comment:
This comments is not related to this file, however it is related to the deleted migrations, change the base of the pull request to practice-lab-1 and you'll see the deleted migrations, I mention this several times in class, deleting migrations is HUGH BAD PRACTICE i know in the current project it doesn't look like that but in a real project with several colaborators this can cause big trouble